esp/esp/program/modules/handlers/jsondatamodule.py

```diff
@@ -468,6 +475,7 @@ def class_info(self, request, tl, one, two, module, extra, prog):
     class_info.cached_function.depend_on_model(ClassSection)
 
     @aux_call
+    @no_auth
```

Comment:
In other cases the decorator went below the `@json_response()`.  Is there a deliberate difference?  Does it matter?